WooCommerce/WooCommerceTests/ViewRelated/Products/AI/ProductSharingMessageGenerationViewModelTests.swift

```diff
@@ -89,4 +89,97 @@ final class ProductSharingMessageGenerationViewModelTests: XCTestCase {
         // Then
         assertEqual(ProductSharingMessageGenerationViewModel.Localization.errorMessage, viewModel.errorMessage)
     }
+
+    // MARK: `shareSheet`
+    func test_shareSheet_has_expected_activityItems() async throws {
+        // Given
+        let expectedString = "Check out this product!"
+        let expectedURLString = "https://example.com"
+        let stores = MockStoresManager(sessionManager: .makeForTesting())
+        let viewModel = ProductSharingMessageGenerationViewModel(siteID: 123,
+                                                                 productName: "Test",
+                                                                 url: expectedURLString,
+                                                                 stores: stores)
+        stores.whenReceivingAction(ofType: ProductAction.self) { action in
+            switch action {
+            case let .generateProductSharingMessage(_, _, _, completion):
+                completion(.success(expectedString))
+            default:
+                return
+            }
+        }
+
+        // When
+        await viewModel.generateShareMessage()
```

Comment:
I don't think we need to generate share message to test the share sheet right? It looks to me that `shareSheet` is a computed variable.

Reply:
`shareSheet` uses `messageContent` to compute its value. And the `messageContent` will be an empty string if we don't generate a share message.